test/parallel/test-string-decoder.js

```diff
@@ -91,7 +91,7 @@ decoder = new StringDecoder('utf8');
 assert.strictEqual(decoder.write(Buffer.from('E1', 'hex')), '');
 
 // A quick test for lastChar, lastNeed & lastTotal which are undocumented.
-assert(decoder.lastChar.equals(new Uint8Array([0xe1, 0, 0, 0])));
+assert(decoder.lastChar.equals(Buffer.from([0xe1, 0, 0, 0])));
```

Comment:
Why this change?

Reply:
To elaborate a bit: There's nothing in this change that breaks `new Uint8Array()` working in this context, so why is the test case being changed instead of a new test case being added?

Reply:
Shouldn't test cases be added for `DataView` and `TypedArray`? Do they even work properly? No code has been added in this PR to alter how they work with `string_decoder`. Were they working already or something?

Reply:
I'm work on the suggestion of [this comment here](https://github.com/nodejs/node/issues/1826#issuecomment-410487110):
> [ ] string_decoder
> This list is for TypedArray/DataView support only.

I was trying not use `Unit8Array`, hopefully I understood the #1826 correctly.

If this `Unit8Array` doesn't need to change to `Butter` to align with others, I can revert this change here. Please help, if I misunderstood the context of string decode in #1826.

Reply:
Buffer extends Uint8Array, I don't think this changed anything.